### ISSUE TITLE
Add word and url preservation

### ISSUE THREAD

**Issue by micjan** [micjan/truncateString#3]
By default the string is cut wherever. Even mid-word or mid-url. With an extra option or two this should be prevented.

**Comment by micjan** [comment]
Add `options.cutAt` array with a list of strings that mark the places in the string to cut.

For example with 

~~~
truncateString('Lorem Ipsum Dolor! Sit Amet.', 8);
~~~

the result would be `Lorem Ip…`. But with

~~~ 
truncateString('Lorem Ipsum Dolor! Sit Amet.', 8, {
  cutAt: [' ']
});
~~~

the result could be `Lorem Ipsum…`, because the cut was moved to the next place where one of the strings in cutAt.

**Comment by micjan** [comment]
Resolved by #9
